src/Task1_S2.java

```diff
@@ -0,0 +1,48 @@
+public class Task1_S2 {
+
+    //    String-2 > doubleChar
+    //    Given a string, return a string where for every char in the original, there are two chars.
+    //    doubleChar("The") → "TThhee"
+    //    doubleChar("AAbb") → "AAAAbbbb"
+    //    doubleChar("Hi-There") → "HHii--TThheerree"
+
+    public static void main(String[] args) {
+        System.out.println(doubleChar("Ukraine"));
+    }
+
+    public static String doubleChar(String str) {
+
+        //Below is my code that runs well on IDEA, but it does not run on https://codingbat.com
+        //showing error "cannot find symbol repeat(int)".
```

Comment:
It's because of the older Java version + you're not allowed to use repeat() for the seek of the learning purposes :)